File: Mobile/app/src/main/java/com/fivegearszerochill/paladins/domain/models/match/LoadoutSelectedModel.kt
package com.fivegearszerochill.paladins.domain.models.match

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class LoadoutSelectedModel (
    @Expose
    @SerializedName("paladinsMatchId")
    val paladinsMatchID: Long,
    @Expose
    @SerializedName("paladinsPlayerId")
    val paladinsPlayerID: Long,
    @Expose
    @SerializedName("paladinsItemId")
    val paladinsItemID: Long,
    @Expose
    @SerializedName("loadoutItemLevel")
    val loadoutItemLevel: Long,
    @Expose
    @SerializedName("loadoutItemName")
    val loadoutItemName: String
)

File: Paladins.Api/Paladins.Api/Paladins.Common/Models/LoadoutSelectedModel.cs
using Paladins.Common.Interfaces.Definitions;

namespace Paladins.Common.Models
{
    public class LoadoutSelectedModel : IModel
    {
        public int PaladinsMatchId { get; set; }
        public int PaladinsPlayerId { get; set; }
        public int PaladinsItemId { get; set; }
        public int LoadoutItemLevel { get; set; }
        public string LoadoutItemName { get; set; }
    }
}
